configure.ac

```diff
@@ -1282,7 +1282,7 @@ fi
 EXTRA_LDFLAGS="$EXTRA_LDFLAGS $PHP_LDFLAGS"
 EXTRA_LDFLAGS_PROGRAM="$EXTRA_LDFLAGS_PROGRAM $PHP_LDFLAGS"
 
-PHP_BUILD_DATE=`date '+%Y-%m-%d'`
+PHP_BUILD_DATE=`date --date="@${SOURCE_DATE_EPOCH:-$(date +%s)}" +%Y-%m-%d`
```

Comment:
A comment here describing where SOURCE_DATE_EPOCH comes from would be helpful I think.

Reply:
Added a comment about SOURCE_DATE_EPOCH